iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/softwareurl/output_transformer/test/data/document_to_software_url.json

```diff
@@ -0,0 +1,7 @@
+{"documentId": "docId1", "softwareUrl": "https://github.com/openaire/iis", "repositoryName": "GitHub", "confidenceLevel": 0.9}
+{"documentId": "docId1", "softwareUrl": "https://github.com/openaire/iis", "repositoryName": "GitHub", "confidenceLevel": null}
+{"documentId": "docId2", "softwareUrl": "https://github.com/openaire/iis", "repositoryName": "GitHub", "confidenceLevel": 0.9}
+{"documentId": "docId2", "softwareUrl": "https://github.com/openaire/primer", "repositoryName": "GitHub", "confidenceLevel": 0.8}
+{"documentId": "docId3", "softwareUrl": "https://github.com/openaire/vipe", "repositoryName": "GitHub", "confidenceLevel": 0.7}
+{"documentId": "docId3", "softwareUrl": "https://github.com/openaire/primer", "repositoryName": "GitHub", "confidenceLevel": null}
+{"documentId": "docId3", "softwareUrl": "https://github.com/openaire/iis", "repositoryName": "GitHub", "confidenceLevel": 0.7}
```

Comment:
I don't think so. `DocumentToSoftwareUrl` derived from other already existing `referenceextraction` schemas where confidenceLevel was defined as:
```
union { null , float } confidenceLevel = null;
```
I guess it will be reasonable to create separate ticket for making `confidenceLevel` mandatory for all `referenceextraction` algorithms. I don't think they can provide null value.